services/workflows-service/src/events/get-webhook-info.ts

```diff
@@ -8,10 +8,11 @@ export const getWebhookInfo = (
   NODE_ENV: string | undefined,
   WEBHOOK_URL: string | undefined,
   WEBHOOK_SECRET: string | undefined,
+  event: string,
```

Comment:
Might be an existing type, could be `TEvent` which is exported from the nearby types file.